test/detect-images.sh

```diff
@@ -8,6 +8,9 @@ pathToTests=$(dirname $0)
 echo "Changed files between $@:"
 for file in $(git --no-pager diff --name-only "$@"); do
 	dir=$(dirname $file)
+	if [ $(basename $dir) = "dockerdir" ]; then
+		dir=$(dirname $dir)
+	fi
```

Comment:
the last thing last, sorry, can we avoid adding these lines, but try the following code in line 10, I think it should work
```
dir=$(dirname $file | cut -d '/' -f 1)
```
I know 😞  I have introduced this bug in line 10

Reply:
I changed it as you suggested - I wasn't sure if that previous check was intentional, so I included only "dockerdir"